parsers/mysql/mysql.go

```diff
@@ -325,38 +331,38 @@ func (p *Parser) handleEvent(rawE []string) (SlowQuery, time.Time) {
 				"line":  line,
 				"event": rawE,
 			}).Debug("readmin ping detected; skipping this event")
-			sq.skipQuery = true
+			return nil, time.Time{}
```

Comment:
it does when it's a nil masquerading as a `map[string]interface{}`! one of the hilariously fun failures of `reflect.DeepEqual` vs `==`.

https://play.golang.org/p/JHTuJsDgLn